code/core/src/builder-manager/index.ts

```diff
@@ -234,7 +238,11 @@ const builder: BuilderFunction = async function* builderGeneratorFn({ startTime,
   yield;
 
   const addonsDir = config.outdir;
-  const coreDirOrigin = join(dirname(require.resolve('@storybook/manager/package.json')), 'dist');
+  const coreDirOrigin = join(
+    dirname(require.resolve('@storybook/core/package.json')),
```

Comment:
Shouldn't we use `storybook/core-path` here?

Reply:
`@storybook/core` cannot depend on `storybook` that would be a cyclical dependency.